### DISCUSSION TITLE
how to run lldpd without creating lldpd user

### DISCUSSION THREAD

**Discussion by raviteja-b** [lldpd/lldpd#671] [Q&A]
I was trying to pull LLDPD package and run lldpd daemon but i see errors while running LLDPD daemon if I dont create lldpd user on system, but I don't want to create new lldp user on my system.

how to configure LLDPD to run without creating new lldpd user?

**Comment by vincentbernat** [comment]
Configure with:
```
    --with-privsep-user=root \
    --with-privsep-group=root \
```

**Comment by raviteja-b** [comment]
@vincentbernat can you please share example lldpd configuration file which sets below configuration
```
 --with-privsep-user=root \
    --with-privsep-group=root \
```

**Reply by vincentbernat** [reply]
It's not a configuration file. You need to recompile lldpd with these options.

**Reply by raviteja-b** [reply]
> It's not a configuration file. You need to recompile lldpd with these options.


@vincentbernat 
I am using bitbake recipe file to pull this lldpd into our yocto builds.
can you please suggest how to configure  these during yocto build
